File: Examples/CSharp/WorkingWithResources/GetResourceCalendar.cs
namespace Aspose.Tasks.Examples.CSharp.WorkingWithResources
{
    using System;

    internal class GetResourceCalendar
    {
        public static void Run()
        {
            var dataDir = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod());

            //ExStart:GetResourceCalendar
            //ExFor: Project.Resources
            //ExSummary: Shows how to read project resources.
            var project = new Project(dataDir + "ResourceCalendar.mpp");

            // Display base calendar name for all resources
            foreach (var resource in project.Resources)
            {
                if (resource.Get(Rsc.Name) != null)
                {
                    Console.WriteLine(resource.Get(Rsc.Calendar).BaseCalendar.Name);
                }
            }
            //ExEnd:GetResourceCalendar
        }
    }
}

File: Examples/VisualBasic/WorkingWithResources/GetResourceCalendar.vb

Namespace WorkingWithResources
    Public Class GetResourceCalendar
        Public Shared Sub Run()
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType.FullName)

            ' ExStart:GetResourceCalendar
            ' Create project instance
            Dim project1 As New Project(dataDir + "ResourceCalendar.mpp")

            ' Display base calendar name for all resources 
            For Each res As Resource In project1.Resources
                If res.Get(Rsc.Name) IsNot Nothing Then
                    Console.WriteLine(res.Get(Rsc.Calendar).BaseCalendar.Name)
                End If
            Next
            ' ExEnd:GetResourceCalendar
        End Sub
    End Class
End Namespace
